superset-frontend/src/dashboard/actions/dashboardInfo.ts

```diff
@@ -177,3 +186,48 @@ export function saveFilterBarOrientation(orientation: FilterBarOrientation) {
     }
   };
 }
+
+export function saveCrossFiltersSetting(crossFiltersEnabled: boolean) {
```

Comment:
This function is pretty similar to `saveFilterBarOrientation`. I'm wondering if we could DRY it a bit 🤔